### PR TITLE
add setup instructions

### PR DESCRIPTION
**Pull request by esteblock**
I added more instructions in README,
Specifically regarding rustc setup needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,21 +11,26 @@ Getting Started
...
-2. If you want to run everything locally: `docker` (you can run both Standalone and Futurenet backends with it)
-3. Node.js v18
-4. [Freighter Wallet](https://www.freighter.app/) ≥[v5.0.2](https://github.com/stellar/freighter/releases/tag/2.9.1). Or from the Firefox / Chrome extension store. Once installed, enable "Experimental Mode" in the settings (gear icon).
-5. If you want to run everything locally, build the `soroban-preview` docker image:
+1. `rustc` >= 1.71.0 with the `wasm32-unknown-unknown` target installed. See https://soroban.stellar.org/docs/getting-started/setup#install-rust . If you have already a lower version, the easiest way to upgrade is to uninstall (`rustup self uninstall`) and install it again.
```
[the remark this answers is not in the document]
Comment (chadoh) [reply README.md:14]: Or `rustup update`, it looks like

[the remark this answers is not in the document]
Comment (esteblock) [reply README.md:14]: Yes, That works. however won''t do all the work as after doing that (at least in my machine) you'll need to do something like
```
rustup use stable-x86_64-unknown-linux-gnu
```

And that depends on each machine... The user shoul check on `rustup show`.

That's why that I just said that the easier way was to uninstall and install again

[the remark this answers is not in the document]
Comment (esteblock) [reply README.md:14]: This is with the aim to reach newcommers to the ecosystem

[the remark this answers is not in the document]
Comment (chadoh) [reply README.md:14]: Ok, I'll take your word for it! I've only ever done `rustup upgrade`

**Review by chadoh — APPROVED** [review]

**Review by 0xAegir — APPROVED** [review]

**Pull request merged**